src/services/study/network-modifications.js

```diff
@@ -1742,6 +1742,39 @@ export function modifyByFormula(studyUuid, currentNodeUuid, equipmentType, formu
     });
 }
 
+export function modifyByAssignment(
+    studyUuid,
+    currentNodeUuid,
+    equipmentType,
+    assignmentsList,
+    isUpdate,
+    modificationUuid
+) {
+    let modificationUrl = getStudyUrlWithNodeUuid(studyUuid, currentNodeUuid) + '/network-modifications';
```

Comment:
This line is copy pasted 35 times. May be a small `getNetworkModifsUrl(studyUuid, currentNodeUuid)` function or whatever. (the following lines are also extremely similar)

Reply:
DONE! create new function getNetworkModificationUrl